data_and_models/pipelines/sentence_embedding/dvc.lock

```diff
@@ -221,7 +221,7 @@ eval_biobert_nli_sts_cord19_v1:
     md5: f80bd92608012c65ede3417b10b0e731
     size: 1264
   - path: ../../metrics/sentence_embedding/biobert_nli_sts_cord19_v1.json
-    md5: c53426897361e4b4c66e58a78c85dad4
+    md5: c4371ca281d5f224332115f5a31ce9cd
```

Comment:
Have the performances of our production model changed then?

Reply:
I think the models predictions haven't changed—Kendall's τ and Spearman's ρ are the same. 
But probably `scipy` slightly changed the implementation of Pearson's r score— which is why we have some negligible difference (`~1e-7`) in the resulting Pearson's r value.

For a full reference, here you have all the changes:
```bash
bbs_292  [c4371ca281d5f224332115f5a31ce9cd] -- {"kendall_tau": 0.5778437302402729, "pearson_r": 0.7254631784119863, "spearman_rho": 0.7167140825467725}
master   [c53426897361e4b4c66e58a78c85dad4] -- {"kendall_tau": 0.5778437302402729, "pearson_r": 0.7254630892864342, "spearman_rho": 0.7167140825467725}
```

Reply:
Interesting. Thanks for the clarification.